pkg/operators/cli/clioperator.go

```diff
@@ -179,6 +179,59 @@ func (o *cliOperatorInstance) ExtraParams(gadgetCtx operators.GadgetContext) api
 	return api.Params{fields, mode}
 }
 
+func parseFields(fieldsString string, defaultFields []string) ([]string, error) {
+    fields := strings.Split(fieldsString, ",")
+    showFields := make(map[string]struct{})
+    hideFields := make(map[string]struct{})
+    addedFields := make(map[string]struct{})
+    
+    // Initially, all default fields are shown
+    for _, field := range defaultFields {
+        showFields[field] = struct{}{}
+    }
+
+    for _, field := range fields {
+        field = strings.TrimSpace(field)
+        if field == "" {
+            continue
+        }
+        switch field[0] {
+        case '+':
+            fieldName := field[1:]
+            if _, ok := hideFields[fieldName]; ok {
+                return nil, fmt.Errorf("field %q both added (+) and removed (-)", fieldName)
+            }
+            addedFields[fieldName] = struct{}{}
+        case '-':
+            fieldName := field[1:]
+            if _, ok := addedFields[fieldName]; ok {
+                return nil, fmt.Errorf("field %q both added (+) and removed (-)", fieldName)
+            }
+            hideFields[fieldName] = struct{}{}
+        default:
+            // If no prefix, return only these fields
+            return strings.Split(fieldsString, ","), nil
```

Comment:
This would fail if something like `--fields=timestamp,+error` is passed. I think in that case the timestamp and error columns should be shown.

Reply:
Hi @mauriciovasquezbernal, sorry for late changes, involved in creating LFX proposal. I've made changes, 
It uses, `slices.DeleteFunc` for remove fields. Also added `fields=timestamp,+error` case.